docker/scripts/setup-pbm.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
This file name seems to be a bit misleading IMO, since it actually performs a backup. Can we rename it to, let's say, `pbm-backup.sh`?

Reply:
Yeah the name was from it setting up pbm containers for us to get metrics (so it includes allowing access to storage, configuring pbm, and taking the actual backup).